### ISSUE TITLE
GET /Users endpoint without a filter will be deprecated

### ISSUE THREAD

**Issue by arianvp** [wireapp/robin#143]
https://github.com/wireapp/robin/blob/6a83fdda0dd12c8b7e14613f219a7e195bb33564/src/main/java/com/wire/bots/robin/ScimClient.java#L131

Instead; `GET /Users` will need an explicit query parameter that makes it return a unique user.

1.  `GET /Users?filter=externalId eq <email address>`
2.   `GET /Users?filter=userName eq <wire-handle>`

**Comment by dkovacevic** [comment]
Are those endpoint already available in prod?

**Comment by arianvp** [comment]
Nope;  in staging by the end of tomorrow; on prod by the end of wednesday

**Comment by arianvp** [comment]
These endpoints are in production.

@akshaymankar is going to need to remove this endpoint soon-ish as part of the large-teams feature

**Comment by akshaymankar** [comment]
Here is the issue: https://github.com/zinfra/backend-issues/issues/1186
